docs/security/password-protection.mdx

```diff
@@ -17,15 +17,39 @@ While these rules might seem lax independently, NIST's additional leaked passwor
 
-## Leaked password protection
-Clerk refers to the National Institute of Standards and Technology (NIST) guidelines to determine its handling of leaked passwords:
+## Reject compromised passwords
+Clerk refers to the National Institute of Standards and Technology (NIST) guidelines to determine its handling of compromised passwords:
 
 When processing requests to establish and change memorized secrets, verifiers SHALL compare the prospective secrets against a list that contains values known to be commonly-used, expected, or compromised. For example, the list MAY include, but is not limited to: [NIST Special Publication 800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html#sec5)
 
 - Passwords obtained from previous breach corpuses.
 
 Specifically, Clerk contracts with have [i been pwned](https://haveibeenpwned.com/) to compare prospective passwords against its corpus of over 10 billion leaked credentials.
 
+Rejection of compromised passwords is enabled by default on all Clerk apps, for sign-up and password changes.
+
+Settings for this feature can be found in the Clerk Dashboard under **User & Authentication > [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)**, in the **Authentication strategies > Password** section.
+
+![Settings for compromised password handling](/public/images/authentication/reject_compromised_passwords.png)
+
+### Reject compromised passwords on sign-in
+As can be seen in the above screenshot, Clerk also offers the capability to check for compromised passwords upon sign-in.
+
+This is useful for blocking password sign-ins in the case that:
+
+* the password has recently been added to the compromised password database
+* the user was able to set a compromised password because protection was off at the time
+* the user was migrated to Clerk along with their existing password digest
+
+If enabled, when the user provides their correct password that nevertheless has been found in online breach data, they will not be signed in, but instead prompted to reset their password.
+
+Password reset for compromised passwords utilizes the same flow as for "forgot password", i.e. the user will need to authenticate first via an OTP code on their email or phone and only then they will be able to set a new, more secure, password.
+
+**Disclaimers:**
+
+* Prior to enabling rejection of compromised passwords on sign-in, please ensure your app has support for the password reset flow, either by utilizing Clerk components or via your [own custom implementation](/docs/custom-flows/forgot-password)
+* If the user has no way of resetting their password, e.g. if the application configuration allows for a user to be without a communication method, then Clerk will not enforce this restriction for the impacted user
```

Comment:
```suggestion
- If the user has no way of resetting their password, e.g. if your application does not require an email, phone number, or other communication method to sign up, Clerk will not reject compromised passwords on sign-in.
```

Reply:
Remove the use of "e.g." as it might not be understood in other languages/cultures.

```suggestion
- If the user has no way of resetting their password—such as when your application does not require an email, phone number, or other communication method for sign-up—Clerk will not reject compromised passwords on sign-in.
```